src/beautifiers/cljfmt/index.coffee

```diff
@@ -13,9 +13,8 @@ module.exports = class Cljfmt extends Beautifier
 
   beautify: (text, language, options) ->
     formatPath = path.resolve(__dirname, "fmt.edn")
-    cljfmt = path.resolve(__dirname, "..", "..", "..", "node_modules/.bin/cljfmt")
     @tempFile("input", text).then((filePath) =>
-      @run(cljfmt, [
+      @run("cljfmt", [
```

Comment:
I've changed my mind slightly. I would prefer this to be backwards compatible and opt-in for using the globally installed `cljfmt`. So by default would use the `cljfmt` already installed within `node_modules/` and then users can use their own version by manually configuring the path.